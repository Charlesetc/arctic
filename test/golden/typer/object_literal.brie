define main [ <x = 3, y = 4> ]
