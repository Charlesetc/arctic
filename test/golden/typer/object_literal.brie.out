{[0, 2]=>Literal(subtype=integer), [1, 3]=>Literal(subtype=integer), [4]=>Closed_object(fields={"x"=>3, "y"=>2})}